contract/src/stage1/function_call.rs

```diff
@@ -23,50 +25,66 @@ impl Keypom {
         account_id: AccountId,
         funder_id: AccountId,
         drop_id: DropId,
+        fc_args: Option<Vec<Option<String>>>,
     ) {
         /*
             Function Calls
         */
         let gas = fc_config.and_then(|c| c.attached_gas).unwrap_or(Gas(0));
 
-        for method in methods {
-            let keypom_args = KeypomArgs {
-                account_id_field: method.account_id_field.clone(),
-                drop_id_field: method.drop_id_field.clone(),
-                key_id_field: method.key_id_field.clone(),
-                funder_id_field: method.funder_id_field.clone(),
-            };
-
+        for (i, method) in methods.iter().enumerate() {
             let mut final_args = method.args.clone();
-
-            if final_args.contains("\"keypom_args\"") {
+            if final_args.len() == 0 {
+                final_args = "{}".to_string();
+            }
+            
+            let try_json: Result<Value, _> = from_str(&final_args);
+            if try_json.is_err() {
                 near_sdk::log!(
-                    "Keypom Args detected in client args. Returning and decrementing keys"
+                    "Cannot cast args to JSON. Returning and decrementing keys"
```

Comment:
Should we tell them decrementing keys effect? Like do we decrement uses, gas, both, etc...?